docs/Server-Methods.md

```diff
@@ -143,6 +143,10 @@ The logger instance, check [here](https://github.com/fastify/fastify/blob/master
 #### inject
 Fake http injection (for testing purposes) [here](https://github.com/fastify/fastify/blob/master/docs/Testing.md#inject).
 
+<a name="add-schema"></a>
+#### addSchema
+`fastify.addSchema(schemaObj)`, adds a shared schema to the Fastify instance, then you can reuse it everywhere in your application just by writing the schema id that you need. You can see [here](https://github.com/fastify/fastify/blob/master/docs/Validation-and-Serialization.md#shared-schema) how it works.
```

Comment:
I don't know if "everywhere" is correct here.
From developer point of view, "everywhere" *may* include the serialization schema.

Reply:
As I wrote [here](https://github.com/fastify/fastify/pull/735#issuecomment-362239385):

> the code works for everything is inside the opts.schema object, if in the future we will add a foo key, the code will just work™ ;)

Add schema will work for **every** schema you declare.